sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -1085,3 +1074,25 @@ func withDynamicResolver(ctx context.Context, ref string, registryConfig *Regist
 		return defaultResolver
 	}
 }
+
+// withMaskedPaths sets an alternate list of masked paths
+func withMaskedPaths() oci.SpecOpts {
+	return func(_ context.Context, _ oci.Client, _ *containers.Container, s *runtimespec.Spec) error {
+		if s.Linux != nil && s.Linux.MaskedPaths != nil {
+			// Paths we want to unmask
+			unmaskPaths := []string{
+				"/sys/firmware",
+			}
+			var maskedPaths []string
+			for _, path := range s.Linux.MaskedPaths {
+				if slices.Contains(unmaskPaths, path) {
```

Comment:
If all we need `k8s.io/utils` for is a string slice checker, I'm in favor of dropping that dependency and writing it ourselves:

```diff
-                if slices.Contains(unmaskPaths, path) {
-                         continue
-                 }
+                for _, unmaskedPath := range unmaskedPaths {
+                    if unmaskedPath == path {
+                        continue
+                    }
+                }
```

Because [a little copying is better than a little dependency.](https://youtu.be/PAAkCSZUG1c?t=569)

Reply:
I'm not opposed to this, but I guess I'm wondering if we should just skip the `[]string` altogether since we only have one path, and switch back if that is ever not the case (which it may never be).

Reply:
I think handling a slice of length 1 is fine. And it makes the code abit more extensible too. I also like @arnaldo2792's suggestion of making the function signature:
```go
func withMaskedPaths(unmaskedPaths []string) oci.SpecOpts {
```
so then we can call this function like:
```go
WithMaskedPaths([]string{"/sys/firmware"})
```
and the function _body itself_ doesn't have any assumptions. Any changes in the future would just be changes to the arguments of a function, not the function itself.

But, I'm getting philosophic again, so it's up to you on how you'd like to handle the actual implementation, I just think removing `k8s.io/utils` is wise.